shacl12-core/index.html

```diff
@@ -2920,8 +2920,9 @@ <h3>Validation Report</h3>
 				<p>
 					The <dfn data-lt="validation report|report">validation report</dfn> is the result of the <a>validation</a> process that reports the <a>conformance</a> and the set of all <dfn data-lt="validation result|result">validation results</dfn>.
 					The validation report is described with the SHACL <dfn>Validation Report Vocabulary</dfn> as defined in this section.
-					This vocabulary defines the RDF properties to represent structural information that may provide guidance on how to identify or fix violations in the data graph.
+					This vocabulary defines the RDF classes and properties to represent structural information that may provide guidance on how to identify or fix violations in the data graph:
 				</p>
+				<div id="ValidationReport-UML.svg></div>
```

Comment:
```suggestion
				<div id="ValidationReport-UML.svg"></div>
```

I believe this was the reason why the UML did not show.